.templates/docs.sh.template

```diff
@@ -1,101 +0,0 @@
-        conda activate travis-ci-"$TRAVIS_JOB_NUMBER"
-        exe git clone -b $BRANCH_NAME https://github.com/{{ repo_name }}.git nengo-fpga-"$TRAVIS_JOB_NUMBER"
-
-        # The docs_dir is set in the base and has "../" prepended,
-        # so we enter another dir first
-        cd nengo-fpga-"$TRAVIS_JOB_NUMBER"
-        {{ super() | indent(4) }}
-        exit "\$STATUS"
-EOF
-
-{% endblock %}
-
-{% block script %}
-    # shellcheck disable=SC2034
-    exe ssh fpga-ci << EOF
-        export STATUS=0  # Status flag for remote
-        $(typeset -f exe)  # Send definition of exe function
-        conda activate travis-ci-"$TRAVIS_JOB_NUMBER"
-        cd nengo-fpga-"$TRAVIS_JOB_NUMBER"
-
-        FPGA_CONFIG="\$HOME/fpga_config"
-
-        # A valid fpga_config should be in the $HOME dir of the remote machine
-        if [ ! -f "\$FPGA_CONFIG" ]; then
-          echo "ERROR: There must be a valid fpga_config file in \$HOME"
-          exit 1
-        fi
-
-        cp -f "\$FPGA_CONFIG" .  # Grab config so the system can use the local FPGAs
-        {{ super() | indent(4) }}
-        exit "\$STATUS"
-EOF
-{% endblock %}
-
-{% block after_success %}
-    # shellcheck disable=SC2034
-    exe ssh fpga-ci << EOF
-        export STATUS=0  # Status flag for remote
-        $(typeset -f exe)  # Send definition of exe function
-        # The docs_dir is set in the base and has "../" prepended,
-        # so we enter another dir first
-        cd nengo-fpga-"$TRAVIS_JOB_NUMBER"
-
-        # Taken from docs.sh template after_script block (2019-10-07)
-        cd {{ docs_dir }} || exit
-        git config --global user.email "travis@travis-ci.org"
-        git config --global user.name "TravisCI"
-        git add --all
-
-        if [[ "$TRAVIS_BRANCH" == "$TRAVIS_TAG" ]]; then
-            exe git commit -m "Documentation for release $TRAVIS_TAG"
-            exe git push -q "https://$GH_TOKEN@github.com/{{ repo_name }}.git" gh-pages-release
-        elif [[ "$BRANCH_NAME" == "master" ]]; then
-            exe git commit -m "Last update at $(date '+%Y-%m-%d %T')"
-            exe git push -fq "https://$GH_TOKEN@github.com/{{ repo_name }}.git" gh-pages-release:gh-pages
-        elif [[ "$TRAVIS_PULL_REQUEST" == "false" ]]; then
-            exe git commit -m "Documentation for branch $TRAVIS_BRANCH"
-            exe git push -fq "https://$GH_TOKEN@github.com/{{ repo_name }}.git" gh-pages-release:gh-pages-test
-        fi
-        exit "\$STATUS"
-EOF
-{% endblock %}
-
-{% block after_script %}
-    # Cleanup since we are on a physical machine (abrhost)
-    exe ssh fpga-ci << EOF
-        # The docs_dir is set in the base and has "../" prepended,
-        # so we enter another dir first
-        cd nengo-fpga-"$TRAVIS_JOB_NUMBER"
-        rm -rf {{ docs_dir }}
-        cd ..
-        rm -rf nengo-fpga-"$TRAVIS_JOB_NUMBER"
-        conda env remove -y -n travis-ci-"$TRAVIS_JOB_NUMBER"
```

Comment:
I don't see this in the new generated script... Is this handled differently now?

Reply:
No I think we should have this in there, I just neglected to add this in since I was relying on the templated scripts, but those are for VMs so we definitely want to put this back in :+1:

Do we want to omit the miniconda install pieces too? I think it would be cleaner to leave that in so we don't need to modify the base tempalte as much... not sure if there are any adverse effects of it grabbing the installer and trying to install again?

Reply:
If we leave the miniconda install part in there, where does it install it to? Does it keep installing additional copies of miniconda every time the script runs? 

Also, I'm not convinced putting in the conda cleanup should be in our scripts. I think it might be more an nengo-bones issue... It should do proper cleanup, VM or not.

Reply:
As an example, the stuff being run on the azure cloud is a VM, but a persistent VM (i think), so you would still want the conda cleanup there.

Reply:
It is cleaned up, here https://github.com/nengo/nengo-bones/blob/master/nengo_bones/templates/remote.sh.template#L105